vendor/licenses/git_submodule/atom-language-stan/stan-language-definitions.dep.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: stan-language-definitions
+version: 61c22f919aae181414bb9b79efc9640035c38905
+type: git_submodule
+homepage: https://github.com/jrnold/stan-language-definitions.git
+license: none
+licenses: []
+notices: []
```

Comment:
Whoops. This file isn't related at all.

Reply:
I must have included this in a commit by mistake but I'm not sure how to remove it.

Reply:
If you add me as a collaborator on your repo, or I'll be able to the needed fixes.